integration/apocalypse/gethnet/apocalypse.json

```diff
@@ -0,0 +1,819 @@
+{
```

Comment:
The genesis templates in this PR are used to generate the concrete genesis files.  The `config` command takes the templates and adds the `personas` to them so that it can pre-fund their accounts.